index.html

```diff
@@ -2498,8 +2516,32 @@ <h3>Resize to width</h3>
         <p class="link"></p>
       </div>
       <!-- ends Resize to width -->
-
     </div>
+        <div class="well">
+      <h2 id="flac">flac</h2>
+
+      <!-- flac tool -->
```

Comment:
I think you meant to have a flac section with these three sections nested beneath, but this just opens one section with all the recipes

Reply:
The one section is as intended - transcode and decode seemed similar and connected enough not to merit two separate recipes. Also, since I don't really see this as an extensible/growing section (like ImageMagick) it seemed most logical to include the about along side the single command, especially regarding not clogging up the ToC. Which I forgot to add it to hahaha